rpc/client/local/local.go

```diff
@@ -49,9 +49,9 @@ type Local struct {
 // you can only have one node per process.  So make sure test cases
 // don't run in parallel, or try to simulate an entire network in
 // one process...
-func New(node *nm.Node) *Local {
+func New(node nm.NodeInterface) *Local {
```

Comment:
Is this cause you would like to reuse the RPC in optimint?

Reply:
I think that by reusing RPC have following pros:
 - easy integration with cosmos-sdk (see lazyledger/cosmos-sdk#20)
 - don't need to reinvent the wheel

Do you see any cons?

Reply:
I agree with you, dont see any cons outside the known issues within the RPC layer will be known issues in optimint. They mainly stem from event streaming and mutex contention